### PR TITLE
Update 3

### PR DESCRIPTION
**Pull request by Rainb0wKey**
ok

### CODE REVIEW

**Comment by Rainb0wKey** [comment]
Please correct the argument description in `$and[]`

File: src/bdscript/argCount.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-## Usage
+## Syntax
 ```
 $argCount[text]
 ```
 
-### Breakdown
-- `text` - The text to get word count from.
+## Parameters
+- `text`: The text to get word count from.
```
Comment (MineBartekSA) [inline src/bdscript/argCount.md]:
```suggestion
- `text`: The text to get word count for.
```

File: src/bdscript/and.md [hunk 2]
```diff
@@ -1,16 +1,35 @@
...
+Returns `true` if every provided condition is true, otherwise `false` is returned.
 
-## Usage
+## Syntax
 ```
-$and[condition1;condition2;etc]
+$and[conditions;...]
 ```
+## Parameters
+- `conditions`: All checks that will be carried out to obtain the desired result. **( didn't know what to write, please change it )**
```
Comment (MineBartekSA) [inline src/bdscript/and.md]: This should sufficiently explain the parameter.
```suggestion
- `conditions`: Checks that will be carried out. All conditions must be true for this function to return `true`. Separate conditions using `;`.
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

**Review by MineBartekSA — APPROVED** [review]
Thank you for your hard work

**Pull request merged**